### ISSUE TITLE
Annotations not working as expected in vanilla js 

### ISSUE THREAD

**Issue by sergioalcalaweb** [Lattice-Automation/seqviz#87]
Hello 

I have this code and i only want to click on annotation in linear or circular graph to get all information but the response is not the same as describe in the readme file

Any ideas?

Thank you very much in advance.

```html
<!doctype html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <script type="text/javascript"
        src="https://cdn.jsdelivr.net/npm/regenerator-runtime@0.13.3/runtime.min.js"></script>
    <script type="text/javascript" src="https://unpkg.com/seqviz"></script>
</head>

<body>
    <div id="root"></div>
    <script>

      window.seqviz
      .Viewer("root", {
          name: "L09136",
          seq: "tcgcgcgtttcggtgatgacggtgaaaacctctgacacatgca",
          style: { height: "100vh", width: "100vw" },
          onSelection: selection => console.log(selection),
          annotations: [
          {
            "start": 0,
            "end": 10,
            "direction": 1,
            "name": "RBS",
            "type": "ANNOTATION",
            "color": "#80D849"
          }
        ]
      }).render();
    </script>
</body>

</html>
```
# output 

```javascript 
  {
    clockwise: true,
    element: undefined,
    end: 7,
    gc: 0,
    length: 0,
    ref: null,
    seq: "",
    start: 7,
    tm: 0,
    type: ""
  }

**Comment by jjti** [comment]
Sorry about that, you're right, it was bugged. I found that we weren't using the parsed annotations correctly (with assigned ids). I made a fix and it now works (eg below). I'll let you know once a change is live

```
{ref: "6998515", seq: "tcgcgcgttt", gc: 60, tm: 32, type: "ANNOTATION", …}
clockwise: true
element: SeqBlock {props: {…}, context: {…}, refs: {…}, updater: {…}, componentWillUnmount: ƒ, …}
end: 10
gc: 60
length: 10
ref: "6998515"
seq: "tcgcgcgttt"
start: 0
tm: 32
type: "ANNOTATION"
__proto__: Object
```

**Comment by jjti** [comment]
this should be fixed in 3.0.27. Feel free to re-open if I'm wrong or the issue persists: https://www.npmjs.com/package/seqviz
